Complete support colons in facet values

cintek opened the pull request:
I'm sorry but somehow I missed one line in the code in #1395 where it is also necessary to replace the parameter for `split`.

hugoeanogueira commented:
I didn't check the code thoroughly, but when applying your patch, I noticed a similar block in line 249.
Do you think this needs a fix too?

```ts
} else if (typeof filter === 'string') {
  const [facet, value] = filter.split(':')

  const facetFilterConfig = facetFilterMap[facet]
```

cintek commented:
@hugoeanogueira thanks for testing it.

> I noticed a similar block in line 249. Do you think this needs a fix too?
> 
> ```ts
> } else if (typeof filter === 'string') {
>   const [facet, value] = filter.split(':')
> 
>   const facetFilterConfig = facetFilterMap[facet]
> ```

I added the regex there as well and tested it by hardcoding the variable `filter`.  It works.

> I didn't check the code thoroughly, but when applying your patch, 

This was the last line of code. Other blocks where `split(':')` is used are doing different things.
